source/features/open-issue-to-latest-comment.tsx

```diff
@@ -12,9 +12,9 @@ function init(): void {
 }
 
 function initDashboard(): void {
-	for (const icon of select.all('.js-recent-activity-container :not(a) > .octicon-comment')) {
+	for (const icon of select.all('.js-recent-activity-container :not(.rgh-latest-comment) .octicon-comment')) {
```

Comment:
This is a common confusion about how `:not()` works

This selector does **not** mean
> select `.octicon-comment` as long as it's not inside `.rgh-latest-comment`.
It means: 
> select `.octicon-comment` as long as one of its ancestors doesn't have that class.

There's a number of elements between `.js-recent-activity-container` and `.octicon-comment` so this will always match it. 

`>` limited the search just to `.octicon-comment` so it worked before this last commit.